### PR TITLE
Fixes #12185 - host created via reports has downcased name

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/services/report_importer.rb [hunk 1]
```diff
@@ -64,9 +64,10 @@ def name
   end
 
   def host
-    @host ||= Host::Base.find_by_certname(name) ||
-      Host::Base.find_by_name(name) ||
-      Host::Managed.new(:name => name)
+    hostname = name.downcase
+    @host ||= Host::Base.find_by_certname(hostname) ||
+      Host::Base.find_by_name(hostname) ||
```
Comment (unorthodoxgeek) [inline app/services/report_importer.rb:69]: where(:name => hostname).first instead of find_by_name please
same for find_by_certname

Comment (dLobatog) [inline app/services/report_importer.rb:69]: If this is because of Rails 4, it's fine. `find_by_` is not deprecated, all other finders are.

**Comment by dLobatog** [comment]
:+1: by me, waiting on @unorthodoxgeek feedback on the `find_by` issue before merging.

**Comment by dLobatog** [comment]
@unorthodoxgeek ping? I'll merge tomorrow unless you see other issues - tested the finders on Rails 4 without issue

**Comment by unorthodoxgeek** [comment]
Sorry, didn't work today. Good to merge

**Comment by dLobatog** [comment]
Merged as 514f21cd9426a8824d3820d634cc4879d0edccf6, thanks @ares!

**Pull request closed without merging**